<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientCrawler</class>
 <widget class="QWidget" name="ClientCrawler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientCrawler</string>
  </property>
  <widget class="QSpinBox" name="spinBoxPortServeur">
   <property name="geometry">
    <rect>
     <x>271</x>
     <y>441</y>
     <width>56</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="value">
    <number>8888</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonQuitter">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>449</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAdresseServeur">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>420</y>
     <width>122</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Adresse du serveur : </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNumeroPort">
   <property name="geometry">
    <rect>
     <x>271</x>
     <y>420</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero de port :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAdresseServeur">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>441</y>
     <width>125</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConnexion">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>420</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Connexion</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
